tools/generate_opcodes_all.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env sh
+
+# This script generates internal/vm/opcodes/all.go.
+
+set -e
+
+temp_file=$(mktemp)
+
+(
+	# shellcheck disable=SC2016
+	echo '// Generated by generate_opcodes_all.sh. Run `make fix` to update. DO NOT EDIT.'
+	echo
+	echo 'package opcodes'
+	echo
+	echo 'import "github.com/git-town/git-town/v19/internal/vm/shared"'
+	echo
+	echo '// All provides all existing opcodes.'
+	echo '// This is used to iterate all opcode types.'
+	echo 'func All() []shared.Opcode {'
+	echo '	return []shared.Opcode{'
+	cat internal/vm/opcodes/*.go | grep '^type .* struct .*$' | sed -e 's/type /\t\t\&/' -e 's/ .*/{},/'
```

Comment:
```suggestion
	cat internal/vm/opcodes/*.go | grep '^type .* struct .*$' | sed -e 's/type /\t\t\&/' -e 's/ .*/{},/' | sort
```

On my machine it generates the opcodes in a non-alphabetic order. Sorting them eliminates this problem.

![image](https://github.com/user-attachments/assets/0deb4d1d-c8d7-447e-8048-973e27824b43)

Reply:
Done.

Reply:
Turns out, Ubuntu in the dev container and Ubuntu on GitHub Actions `sort` differently! I generated the file first in the dev container, but when I run the same script from Terminal.app, I get the same result as GitHub and as you.

I added an option to use the most basic collation (`LC_COLLATION=C sort`) which makes them behave the same way.